docs/Install.md

```diff
@@ -3,64 +3,61 @@
 
 The following will walk through the deployment of Pelorus.
 
-## Prerequisites
-
-Before deploying the tooling, you must have the following prepared
-
+**Prerequisites**
+The following are required to deploy Pelorus:
 * An OpenShift 4.7 or higher Environment
 * A machine from which to run the install (usually your laptop)
   * The OpenShift Command Line Tool (oc)
   * [Helm3](https://github.com/helm/helm/releases)
-  * jq
-  * git
+  * JQ
```

Comment:
it's actually jq and git CLIs there is no such CLI as JQ on the system.

Reply:
please lower case JQ to jq